reference/filter-reference.md

```diff
@@ -700,7 +700,7 @@ When multiple `Filter`s are specified in a rule:
  * `onAllow` identifies what to do when the filter returns a
    "modification to the HTTP request":
    - `"break"`: Apply the modification to the request, then end filter
-     processing, and forward the modified request to the upstream
+     processing, and forwards the modified request to the upstream
```

Comment:
This change is in the passive voice, but the rest of the sentence is in the active voice? I'm not sure why you changed this.